pkg/logs/input/container/docker.go

```diff
@@ -42,6 +47,7 @@ type DockerTailer struct {
 	cli           *client.Client
 	source        *config.LogSource
 	containerTags []string
+	logsMetadata  *config.LogsConfig
```

Comment:
It seems like an overkill to me. You don't want to embed the whole LogsConfig if you only use `source` and `service`. Could we have a dedicated struct ?

Reply:
I am not sure about this, I think a user could enable the `multiline` from here, WDYT ? Maybe @NBParis has a strong opinion

Reply:
We need to be able to specified processing rule here include, exclude, multiline and scrubbing.